src/components/blocks/layout/RichText.tsx

```diff
@@ -0,0 +1,36 @@
+import type { Template } from 'tinacms'
+import { TinaMarkdown } from 'tinacms/dist/rich-text'
+import { PageBlocksRichText } from '~tina/__generated__/types'
+import {
+	tinaMarkdownComponents,
+	tinaMarkdownComponentsRichTextTemplate,
+} from '../../fields/TinaMarkdownComponents/TinaMarkdownComponents'
+
+export const RichText = ({ data }: { data: PageBlocksRichText }) => {
+	const { richTextContent } = data
+	return (
+		<div className='w-full flex ustify-center items-center p-10'>
+			<div className='w-full max-w-7xl'>
+				<div className='w-full flex flex-col md:flex-row gap-10 justify-center'>
+					<div className='w-full md:w-1/2 flex flex-col grow justify-center prose prose-headings:my-6 prose-p:my-2'>
+						<TinaMarkdown components={tinaMarkdownComponents} content={richTextContent} />
+					</div>
+				</div>
+			</div>
+		</div>
+	)
+}
+
+export const richTextTemplate: Template = {
```

Comment:
TineMarkdown question. Does the rich text component let you choose the text color? Or will it always be black?

Reply:
By default, TinaMarkdownComponent uses black-colored text as markdown itself doesn't support color (only with inline html). I looked for a bit and don't think tinacms supports changing the color of the markdown natively which is why we need to have the embedded component that can be used inside of the markdown for it.